routes/account.js

```diff
@@ -80,6 +83,86 @@ router.post('/logout', function(req, res) {
     res.sendStatus(200);
 });
 
+router.get('/updateProfile', verifyJWTCookie, function(req, res) {
+    UserModel.findOne({id: req.user.id}, function(err, user) {
+        if (err) {
+            return res.status(422).json( {success: false});
+        }
+        if (user) {
+            res.render('updateProfile', {username: user.username, 
```

Comment:
Nothing in the object that gets passed to the render function is used in updateProfile.mustache. Unless we are displaying the user's information on the page (which maybe we should do just so they know whose account they are updating), we do not need to query the database in this route.

Reply:
I'll get around to adding their information onto the page as well. I just wanted to add the basic functionality of the updateProfile page first.

Reply:
Oh ok sounds good.